@xen-orchestra/web-core/lib/components/ui/dropdown-button/UiDropdownButton.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v3 -->
+<template>
+  <button type="button" class="ui-dropdown-item" :class="{ selected: selected }" :disabled="isDisabled">
+    <VtsIcon :icon="leftIcon" accent="current" class="left-icon" fixed-width />
+    <span class="typo p1-regular label">
+      <slot />
+    </span>
+    <VtsIcon :icon="faAngleDown" accent="current" class="right-icon" fixed-width />
+  </button>
+</template>
+
+<script lang="ts" setup>
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import { useContext } from '@core/composables/context.composable'
+import { DisabledContext } from '@core/context'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons'
+
+const props = defineProps<{
+  disabled?: boolean
+  selected?: boolean
+  leftIcon?: IconDefinition
+}>()
```

Comment:
Because `useContext()` is used on line 25, and the way Vue handles optional boolean props, in this case it is necessary to set the `disabled` prop value as `undefined` by default:
```suggestion
const props = withDefaults(
  defineProps<{
    disabled?: boolean
    selected?: boolean
    leftIcon?: IconDefinition
  }>(),
  { disabled: undefined }
)
```
FYI: https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/web-core/docs/composables/context.composable.md#caveats-boolean-props

Reply:
oh ok